modules/vba_documents/lib/vba_documents/multipart_parser.rb

```diff
@@ -63,12 +64,13 @@ def self.create_file_from_base64(infile)
         contents = content.sub %r{data:((multipart)/.{3,}),}, ''
       end
       decoded_data = Base64.decode64(contents)
+      record_sha256(submission, 'base64', decoded_data) if submission
```

Comment:
Does this mean we're recording it twice for a Base64 upload? I think line 73 sends it back up to parse and this time, it's no longer base64 which means it goes to parse_file where we're already recording the sha256

Reply:
We are adding a sha256 for the entire base64 payload, as well as individual content and metadata. when they submit base64, it contains both, so we do the sha on the whole base64, then metadata and content, attachment etc separately